lib/Doctrine/ODM/PHPCR/Mapping/Annotations/TranslatedProperty.php

```diff
@@ -18,13 +18,9 @@
  */
 
 namespace Doctrine\ODM\PHPCR\Mapping\Annotations;
-
-use Doctrine\Common\Annotations\Annotation;
-
 /**
- * @Annotation
- * @Target("PROPERTY")
+ * @deprecated Use Property instead
  */
-final class Field extends TranslatableProperty
+class TranslatedProperty extends Property
```

Comment:
its just git thinking that this is a rename. i delete Field and rename TranslatedProperty to Property, but add this empty class for BC in case some custom code was typechecking TranslatedProperty for something.